browser/main/TopBar/index.js

```diff
@@ -165,7 +129,7 @@ class TopBar extends React.Component {
   }
 
   handleCodeInit () {
-    ee.emit('top:search', this.refs.searchInput.value)
+    ee.emit('top:search', this.refs.searchInput.value || '')
```

Comment:
Why is this `emit` there? With out adding the `|| ''` I've got an error from `CodeEditor.js` because it seems that the value is undefined here.
How can I test this?

Reply:
LGTM. I didn't notice that it can be undefined.